frappe/database/query.py

```diff
@@ -85,6 +85,11 @@ def func_between(key: Field, value: Union[List, Tuple]) -> frappe.qb:
 	return key[slice(*value)]
 
 
+def func_is(key, value):
+	"Wrapper for IS"
+	return key.isnull() if value.lower() == "set" else key.notnull()
```

Comment:
this should be in reverse?
if something is `set`, it should have some value attached to it so, it should query for `not null` 🤔 

This is what the current output is:
![Screenshot 2022-05-11 at 10 57 28 PM](https://user-images.githubusercontent.com/32034600/167910744-88accfff-613f-4025-816e-4ff36c51ad44.png)

This is with reversing the condition ..ie 
```py
key.isnull() if value.lower() != "set" else key.notnull()
```
![Screenshot 2022-05-11 at 10 57 40 PM](https://user-images.githubusercontent.com/32034600/167910881-60f7a7de-8efb-460b-956e-bed3fe0ab97e.png)

#

Also, I kinda like `isnotnull()` instead of `notnull()` (there seems to be a bit of difference between the two for composite values [which I don't think we use in FW anyway] - [ref](https://blog.andreiavram.ro/postgresql-null-composite-type/)) ..kinda feels more "correct" somehow 😆 

This is how it's query looks like
![Screenshot 2022-05-11 at 10 57 49 PM](https://user-images.githubusercontent.com/32034600/167912558-8a6ce103-bc96-42b7-84a4-f43d3edaa001.png)